include/rapidcheck/Gen.hpp

```diff
@@ -17,6 +18,18 @@ Gen<Decay<typename std::result_of<Mapper(T)>::type>> map(Gen<T> gen,
 
 } // namespace gen
 
+// Concept check for types that have method
+// `Shrinkable<T> G::operator()(const Random &, int) const`
+template<typename T, typename G, typename = void>
+struct MakesShrinkable : std::false_type {};
```

Comment:
Wouldn't it be simpler to do:
```
using MakesShrinkable = std::is_convertible<decltype(std::declval<const G>()(std::declval<Random>(), 0)), Shrinkable<T>>;
```

(Save for possible typos on my part)

Reply:
Didn't realize I could do that.  Made both changes.